x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/field_loading_error.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { EuiCallOut, EuiSpacer, EuiButtonIcon } from '@elastic/eui';
+
+interface Props {
+  title: React.ReactNode;
+  body: React.ReactNode;
+  resendRequest: () => void;
+  'data-test-subj'?: string;
+  'aria-label'?: string;
```

Comment:
With the current setup, this button cannot be made accessible without an `aria-label`. Can we make it required?